R/organization_purge.R

```diff
@@ -0,0 +1,31 @@
+#' Purge an organization
+#'
+#' @export
+#'
+#' @param id (character) name or id of the organization
+#' @template key
+#' @template args
+#' @return an empty list on success
+#' @examples \dontrun{
+#' ckanr_setup(url = "https://demo.ckan.org", key=getOption("ckan_demo_key"))
+#'
+#' # create an organization
+#' (res <- organization_create("foobar", title = "Foo bars",
+#'   description = "love foo bars"))
+#'
+#' # delete the organization just created
+#' res$id
+#' organization_delete(id = res$id)
+#' }
```

Comment:
i believe this brace is not matched. did you make sure to run check on this?

Reply:
thanks, ran check this time